android/tutorials/tutorial1/src/test/java/com/uber/rib/root/RootInteractorTest.kt

```diff
@@ -13,24 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.uber.rib.root;
+package com.uber.rib.root
 
-import org.junit.Before;
-import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.junit.Before
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
 
-@SuppressWarnings("NullAway")
-public class RootInteractorTest {
+class RootInteractorTest {
 
-  @Mock RootInteractor.RootPresenter presenter;
-  @Mock RootRouter router;
+    lateinit var presenter: RootInteractor.RootPresenter? = null
```

Comment:
As mentioned in pervious comments, all nullable vars need to be converted to lateinit nonnull val.